README.rst

```diff
@@ -337,6 +337,29 @@ mapping csv file (see also `mtypes_probability_map_config.yaml`_).
         --mtypes-config-path=$DATA/mtypes/mtypes_probability_map_config.yaml \
         --output-dir=data/ccfv2/me-types/
 
+Subdivide excitatory files into pyramidal subtypes
+-----------------------------------------------
+This should run after the ME mapping from Roussel. To run:
+
+make_new_inhib_exc_outside_SSCX.py
+
+Input support files needed:
+
+annotation atlas (with L23 split): brain_regions.nrrd
+
+annotation hierarchy (with L23 split): hierarchy.json
+
+inhibitory: gad67+_density_v3_MMB.nrrd
+
+total neuron: neuron_density_v3_MMB.nrrd
+
+Output:
+
+excitatory nrrd files in output directory
+
+inhibitory minus cortex in output directory
```

Comment:
Should this not be formatted as a shell code block?

Reply:
I plan on moving this code into the main repository, and wrapping it in a CLI, so that should clear up this, and make it more configurable (to a degree).